src/uwp/Microsoft.Build.Net.CoreRuntimeTask/ResourceHandlingTask.cs

```diff
@@ -457,15 +451,18 @@ private List<ResWInfo> ExtractAssemblyResWList(string assemblyFilePath, out bool
                         ms.Seek(0, SeekOrigin.Begin);
                         
                         string resourceFileName = resourceReference.Name.Value.Remove(resourceReference.Name.Value.Length - 10);
-                        if (resourceReference.Name.Value.Equals(frameworkResourcesName, StringComparison.OrdinalIgnoreCase))
+                        bool isFrameworkResource = resourceReference.Name.Value.Equals(frameworkResourcesName, StringComparison.OrdinalIgnoreCase);
```

Comment:
>isFrameworkResource [](start = 29, length = 19)

nit: you can use containsFrameworkResources instead of defining a new variable here and that will safe restoring the value again.